### DISCUSSION TITLE
似乎没有预设对话？

### DISCUSSION THREAD

**Discussion by hiforrest** [lobehub/lobehub#1645] [General]
最近我刚知道 Lobe-chat，第一步自然是尝试把在 NextChat 里熟悉的「面具」搬到 Lobe-chat 的「助手」里去。但是，我注意到一个小问题——在「助手」里好像没法像在 NextChat 那样设置几句预设对话。我觉得给大型模型提供一些对话示例和格式真的挺重要的，不知道是我没找对方法，还是 Lobe-chat 目前确实缺少这个功能呢？

**Comment by arvinxx** [comment]
目前是没做，后续在 #869 支持

**Reply by hiforrest** [reply]
早就布局好了嘛，一切尽在掌握🙌
